mne_bids/tests/test_write.py

```diff
@@ -717,7 +717,7 @@ def test_set(_bids_validate):
                       extra_params=dict(foo='bar'))
 
     with pytest.raises(FileExistsError, match="already exists"):  # noqa: F821
-        write_raw_bids(raw, bids_basename, output_path=output_path,
+        write_raw_bids(raw, bids_basename, bids_root=output_path,
```

Comment:
```suggestion
        write_raw_bids(raw, bids_basename, bids_root=bids_root,
```

why do you still keep some references to `output_path` ?